packages/graphql-transformers-e2e-tests/src/__tests__/SearchableModelTransformerV2.e2e.test.ts

```diff
@@ -149,6 +151,15 @@ test('query for aggregate scalar results', async () => {
     {},
   );
   expect(searchResponse).toBeDefined();
+
+  let searchPosts = searchResponse.data.searchPosts.items;
+  let expectedValue = searchPosts[0].ups;
```

Comment:
We should not be relying on the first result and compare that value. It should be a know value (before hand) like you had it earlier. Can we just wait for all the data to be propagated before running this test

Reply:
Not relying on the first value, running the searchPosts query and calculating MIN value from the results and verifying it against the aggregate result.

I agree that waiting for the data to be propagated is the best solution, will work on it in a separate PR.